pemFioi/blocklyRobot_lib-1.0.0.js

```diff
@@ -3301,7 +3301,42 @@ var getContext = function(display, infos, curLevel) {
          resetItems();
       }
    };
-   
+
+   context.resetDisplay = function() {

```

Comment:
`resetDisplay` is meant to be called during a `context.reset`, and works like this in some other libraries. (Note that the code split between `reset` and `resetDisplay` is not completely properly implemented in all libraries, but will need to be when multiple libraries are used at the same time.)

I would hence name it `redrawDisplay` instead, as this doesn't do a "reset" of the items.